composer.json

```
@@ -94,70 +94,70 @@
         "psr/container": "^2.0",
         "psr/http-message": "^1.0 || ^2.0",
         "psr/log": "^1.1 || 2.0 || ^3.0",
-        "scheb/2fa-backup-code": "^6.0",
-        "scheb/2fa-bundle": "^6.0",
-        "scheb/2fa-trusted-device": "^6.0",
+        "scheb/2fa-backup-code": "^7.0",
+        "scheb/2fa-bundle": "^7.0",
+        "scheb/2fa-trusted-device": "^7.0",
```

Version 7 only supports PHP 8.2, but we still support PHP 8.1. There is no other functional difference for us (see https://github.com/scheb/2fa/blob/7.x/UPGRADE.md#6x-to-7x)

```suggestion
        "scheb/2fa-backup-code": "^6.0 || ^7.0",
        "scheb/2fa-bundle": "^6.0 || ^7.0",
        "scheb/2fa-trusted-device": "^6.0 || ^7.0",
```